src/views/Login/Ledger.js

```diff
@@ -72,153 +58,135 @@ class Ledger extends React.Component {
-                "Ethereum" application.  If you're running on older firmware,
-                make sure the "browser support" option is turned on.  To sign
-                transactions, you'll also need to enable the "contract data"
-                option.`}
-          </P>
-
-          <P>
-            {`If you'd like to use a custom derivation path, you may enter
-                 it below.`}
-          </P>
-
-          <H2>{'Running on HTTP?'}</H2>
-
-          <P>{`To authenticate and sign transactions with a Ledger, Bridge
-              must be serving over HTTPS on localhost. You can do this via the
-              following:`}</P>
-
-          <ol className={'measure-md'}>
-            <li className={'mt-4'}>
-              {'Install'}{' '}
-              <a
-                target={'_blank'}
-                href={'https://github.com/FiloSottile/mkcert'}>
-                {'mkcert'}
-              </a>
-            </li>
-            <li className={'mt-4'}>
-              {'Install a local certificate authority via '}
-              <code>{'mkcert -install'}</code>
-            </li>
-            <li className={'mt-4'}>
-              {'In your '}
-              <code>{'bridge'}</code>
-              {
-                ' directory, generate a certificate valid for localhost via '
-              }{' '}
-              <code>{'mkcert localhost'}</code>
-              {'.'}
-              {'This will produce two files: '}
-              <code>{'localhost.pem'}</code> {', the local certificate, and '}
-              <code>{'localhost-key.pem'}</code>
-              {', its corresponding private key.'}
-            </li>
-            <li className={'mt-4'}>
-              {'Run '}
-              <code>{'python bridge-https.py'}</code>
-            </li>
-          </ol>
-
-          {basePathSelection}
-          {truePathSelection}
-
-          <Button
-            prop-size={'wide lg'}
-            className={'mt-8'}
-            onClick={this.pollDevice}>
-            {'Authenticate →'}
-          </Button>
-
-          <Button
-            className={'mt-8'}
-            prop-size={'wide lg'}
-            disabled={Maybe.Nothing.hasInstance(wallet)}
-            onClick={this.props.loginCompleted}>
-            {'Continue →'}
-          </Button>
-        </Col>
-      </Row>
-    );
-  }
+  return (
+    <View>
+      <H1>{'Authenticate With Your Ledger'}</H1>
+
+      <H2>{'Running on HTTPS?'}</H2>
+
+      <P>
+        {`Connect and authenticate to your Ledger, and then open the
+            "Ethereum" application.  If you're running on older firmware,
+            make sure the "browser support" option is turned on.  To sign
+            transactions, you'll also need to enable the "contract data"
+            option.`}
+      </P>
+
+      <P>
+        {`If you'd like to use a custom derivation path, you may enter
+             it below.`}
+      </P>
+
+      <H2>{'Running on HTTP?'}</H2>
```

Comment:
we can actually detect if the browser is in http or https by checking `location.protocol` if we want to be clever about this screen